docs/source/api_ref_models.rst

```diff
@@ -67,25 +72,25 @@ llama2
 
 All models from the `Llama2 family <https://llama.meta.com/llama2/>`_.
 
-Request Access on `Hugging Face <https://huggingface.co/meta-llama/Llama-2-7b>`__.
+Important: You need to request access on `Hugging Face <https://huggingface.co/meta-llama/Llama-2-7b-hf>`__ before downloading it.
 
-To download the Llama2-7B model:
+To download the Llama2-7B-hf model:
```

Comment:
nit: would drop the hf, most people probably won't care about the checkpoint format
```suggestion
To download the Llama2-7B model:
```

Reply:
lol, i was thinking it was "human feedback" or something like that, since there is 7b and 7b-hf